openslides/utils/settings.py.tpl

```diff
@@ -136,6 +136,8 @@ MEDIA_ROOT = os.path.join(OPENSLIDES_USER_DATA_PATH, 'media', '')
 # https://docs.djangoproject.com/en/1.10/topics/auth/passwords/#module-django.contrib.auth.password_validation
 # AUTH_PASSWORD_VALIDATORS = []
 
+# Disable the last_login field for users.
+DISABLE_LAST_LOGIN_FIELD = True
```

Comment:
True is ok for me here.

Reply:
Ok. Changed to `ENALBE_LAST_LOGIN_FIELD = False` to keep all default settings on `false`